src/chains/ethereum/ethereum/src/miner/miner.ts

```diff
@@ -268,18 +271,14 @@ export default class Miner extends Emittery<{
       // of the storage key and save it to the db along with it's keccak hashed version of
       // the storage key. Why you might ask? So we can reference the raw version in
       // debug_storageRangeAt.
-      const stepListener = (
-        event: InterpreterStep,
-        next: (error?: any, cb?: any) => void
```

Comment:
I don't understand the removal of the next argument, do you have a one liner explanation, or a link you can share?

Reply:
ethereumjs-vm uses https://www.npmjs.com/package/async-eventemitter which  doesn't require the second argument.